### DISCUSSION TITLE
still getting No such file or directory

### DISCUSSION THREAD

**Discussion by notatechperson101** [openai/whisper#1558] [Q&A]
Hello,

I'm not a tech person, but I successfully installed all I needed to install whisper. However,the problem is that I keep getting "No such file or directory". The latest version of ffmpeg is installed correctly( so it's not that). I was wonder if I'm supposed to put the audio file in a certain place. I saw a comment somewhere about putting it in the whisper directory but where exactly is that? In the scripts file of python where whisper is located or just generally where python is located?  Any help would be appreciated.

**Comment by phineas-pta** [comment]
whenever have error, show full command used and full error message, instead of making assumptions

**Comment by snoman666** [comment]
I had same problem. Was getting bunch of python errors on line xxx etc.. What did it for me was installing Chocolatey and then ffmpeg via Choco, although I did have ffmpeg already, but go figure... 
makes no difference where audio file is, as long as you open an admin command prompt in the same dir.
If you try to use the large model, you need >6 Gb vram, or you get CUDA out of memory error.. I can run up to medium with NVIDIA 1660.

**Reply by notatechperson101** [reply]
Ahh. Thank you. It's working. The problem was I wasn't opening command prompt from directory.
